supplementary/devices/taoshan.rst

```diff
@@ -0,0 +1,69 @@
+
+Taoshan - Sony Xperia L
+===========================
+
+The device is quite weak to run Halium. But it should be possible.
+
+Status
+------
+
+Halium
+^^^^^^
+A port for Halium 7.1 is being worked on by `Aayush Gupta <https://github.com/jonnius>`_. Halium 7.1 builds, but does not install using halium-install. Using halium-install from JBB it fails to copy the rootfs.img to /data.
+
+* Is someone already working on Halium for this device? If YES: link to repositories of **device specific files** (usually github repos called "android_device_manufacturer_codename") and **device kernel** (usually a repo beginning with "android\ *kernel*\ ") and name authors if possible (perhaps link to their github profile)
+
+Distributions
+^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+
+   * - Distribution
+     - Device Specific Files
+     - Kernel
+     - What works
+     - What doesn't work
+   * - `taoshan on LineageOS Wiki <https://wiki.lineageos.org/devices/taoshan>`_
+     - `LineageOS/android_device_sony_taoshan <https://github.com/LineageOS/android_device_sony_taoshan>`_
+     - `LineageOS/android_kernel_asus_msm8916 <https://github.com/LineageOS/android_kernel_asus_msm8916>`_ based on v3.4.0
+     - see device page
+     - see device page
+
+
+Kernel & Hardware
+^^^^^^^^^^^^^^^^^
+
+Mainline (v3.4.0 as of writing)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+I don't know if something that is needed for the device is mainline already.
+
+Cyanogemod based kernels (LOS & UBP)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Both **Halium and LOS share a common base** which is Cyanogenmod. However, the Halium kernel was pacthed with Halium-specific edits. Currently some kernel flags and additional patches were used.
```

Comment:
What you're saying is nearly always the case, so I don't think it's needed to be added here